rt-forecast-2/forecast/utils/run-rt-forecast.R

```diff
@@ -55,63 +64,80 @@ run_rt_forecast <- function(deaths,
                                                          warmup = 250, 
                                                          cores = no_cores, 
                                                          chains = ifelse(no_cores <= 4, 4, no_cores)),
-                                        output = c("region", "samples", "summary", "timing"))
+                                        output = c("region", "samples", "summary", "timing"),
+                                        obs = obs_opts(scale = list(mean = 0.01, sd = 0.001)))
   # Run Rt - ORIGINAL -------------------------------------------------------
    if ("original" %in% models) {
+     # Rt options
+     rt_original <- rt
+     rt_original$future <- purrr::map(rt_original$future, ~ "project")
```

Comment:
Nice

Reply:
let me know if you think of a nicer helper function to replicate this and can add to package

Reply:
Hm - is this very generaliseable outside our use case here? I also can't find a very neat way to do this for multiple arguments...